docs/framework/interop/marshaling-data-with-com-interop.md

```diff
@@ -32,30 +30,33 @@ COM interop provides support for both using COM objects from managed code and ex
   
 -   [Type Library Exporter (Tlbexp.exe)](../../../docs/framework/tools/tlbexp-exe-type-library-exporter.md), which produces a COM type library from an assembly and generates a wrapper that performs marshaling during method calls.  
   
- This section describes the processes for customizing interop wrappers when you can (or must) supply the marshaler with additional type information.  
+ The following sections link to topics that describe the processes for customizing interop wrappers when you can (or must) supply the marshaler with additional type information.  
   
 ## In This Section  
- [COM Data Types](http://msdn.microsoft.com/en-us/f93ae35d-a416-4218-8700-c8218cc90061)  
+[How to: Create Wrappers Manually](how-to-create-wrappers-manually.md)   
+Describes how to create a COM wrapper manually in managed source code. 
+ 
+ [How to: Migrate Managed-Code DCOM to WCF](../../../docs/framework/interop/how-to-migrate-managed-code-dcom-to-wcf.md)  
```

Comment:
nit: should this be listed first to match TOC order?

Reply:
@mairaw, yes, the orders should match. I've changed the order in the TOC.